sig-big-data/charter.md

```diff
@@ -0,0 +1,56 @@
+# SIG Big Data Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+The Big Data SIG serves as a community resource for advising big data and data science related software projects on techniques and best practices for integrating with Kubernetes. This SIG also represents the concerns of users from big data communities to Kubernetes for the purposes of driving new features and other enhancements, based on big data use cases.
```

Comment:
This is isn't really a definition of a SIG, b/c there isn't code ownership in k/k.  Instead, it's more of a working group.